protos/core.proto

```diff
@@ -245,6 +245,10 @@ message ColumnDescriptor {
   string expression = 6;
 
   repeated string tags = 7;
+
+  // BigQuery specific settings.
+
+  repeated string bigqueryPolicyTags = 8;
```

Comment:
Do you think it be worth extracting to `BigQueryColumnOptions` with this as a member, similar to how we use `BigQueryOptions` in actions?

Reply:
Yeah Ben and I chatted about this, as we can't currently see any other future bigquery specific things, or other warehouse specific settings at a column level, we felt a prefixed field name made more sense in this case as the columns object is already getting quite deeply nested.